infrastructure/helm/quantumserverless/README.md

```diff
@@ -15,6 +15,8 @@ helm dependency build
 Update values.yaml file. Find and replace the following strings
 
 - **CLIENTSECRET-CHANGEME**: string used as the secret for a OIDC protocol
+- **APISERVERSECRET-CHANGEME**: string used as the secret for a OIDC protocol for apiserver
```

Comment:
```suggestion
```

Reply:
I think the `APISERVERSECRET-CHANGEME` can be removed because doesn't affect to ray (in this PR at least).